### PR TITLE
Docs improvements

### PR DESCRIPTION
**Pull request by matteofigus**
*Description of changes:*
* Fix a frontend link
* More information on how to make API authenticated requests using the CLI

*PR Checklist:*

- [x] Unit tests (and integration tests if applicable) provided
- [x] All tests pass
- [x] Pre-commit checks pass
- [x] Debugging code removed
- [x] If releasing a new version, have you bumped the version in the main CFN template?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/USER_GUIDE.md [hunk 1]
```diff
@@ -279,9 +286,35 @@ To add more users to the application:
...
-User Pool. For more information, consult the [Cognito REST API integration
-guide].
+### Making authenticated API requests
+
+If you need to use the API directly, you will need to authenticate the requests
+using the User Pool. After resetting the password via the UI, you can make
+authenticated requests using the AWS CLI:
+
+1. Note the _CognitoUserPoolId_ and _CognitoUserPoolClientId_ parameters
+   displayed in the _Outputs_ tab for the stack, and take note of cognito user
```
Comment (HieronymusLex) [inline docs/USER_GUIDE.md:296]:
```suggestion
   displayed in the _Outputs_ tab for the stack, and the Cognito user
```

File: docs/USER_GUIDE.md [hunk 2]
```diff
@@ -279,9 +286,35 @@ To add more users to the application:
...
+
+   ```sh
+   aws cognito-idp admin-initiate-auth \
+     --user-pool-id <user-pool-id> \
+     --client-id <user-pool-client-id> \
+     --auth-flow ADMIN_NO_SRP_AUTH \
+     --auth-parameters '{"USERNAME":"<user-email>","PASSWORD":"<your-password"}'
+   ```
+
+3. Take note of the `IdToken` from the previous command. Now you can run an
```
Comment (HieronymusLex) [inline docs/USER_GUIDE.md:308]: 3. Use the `IdToken` generated by the previous command to make an authenticated
    request to the API. The API base URL can be obtained from the _ApiUrl_ in the
    _Outputs_ tab for the CloudFormation stack. For example, the following command
    will show the matches in the deletion queue:
 
   ```sh
   curl <api-url>/v1/queue -H "Authorization: Bearer <id-token>"
   ```

Comment (HieronymusLex) [reply docs/USER_GUIDE.md:308]: couldn't make it a suggestion since it's multi-line :(

**Review by HieronymusLex — CHANGES_REQUESTED** [review]
LGTM, just a couple of small changes

File: docs/USER_GUIDE.md [hunk 3]
```diff
@@ -259,6 +262,8 @@ resources.
 The solution provides a web user interface and a REST API to allow you to
 integrate it in your own applications.
 
+### First access via UI
```
Comment (ctd) [inline docs/USER_GUIDE.md:265]:
```suggestion
### Logging in for the first time
```

I _think_ this terminology is better suited to the customer's point of view.

File: docs/USER_GUIDE.md [hunk 4]
```diff
@@ -12,6 +12,9 @@ Forget solution.
   - [Provisioning Data Access IAM Roles](#provisioning-data-access-iam-roles)
 - [Deploying the Solution](#deploying-the-solution)
 - [Accessing the application](#accessing-the-application)
+  - [First access via UI](#first-access-via-ui)
```
Comment (ctd) [inline docs/USER_GUIDE.md:15]:
```suggestion
  - [Logging in for the first time](#logging-in-for-the-first-time)
```

**Review by ctd — CHANGES_REQUESTED** [review]
This looks really helpful for anyone wanting to use the API. 
There's just a few style/phrasing suggestions from me.

File: docs/USER_GUIDE.md [hunk 5]
```diff
@@ -279,9 +286,35 @@ To add more users to the application:
 4. Use this page to create or manage users. For more information, consult the
    [Managing Users in User Pools Guide].
 
-In order to use the API, you will need to authenticate the requests using the
-User Pool. For more information, consult the [Cognito REST API integration
-guide].
+### Making authenticated API requests
+
+If you need to use the API directly, you will need to authenticate the requests
+using the User Pool. After resetting the password via the UI, you can make
```
[the remark this answers is not in the document]
Comment (matteofigus) [reply docs/USER_GUIDE.md:292]: I think that's true indeed for both admin users and new users

File: docs/USER_GUIDE.md (hunk 2 above)
Comment (matteofigus) [reply docs/USER_GUIDE.md]: Done

File: docs/USER_GUIDE.md [hunk 6]
```diff
@@ -279,9 +286,35 @@ To add more users to the application:
...
+authenticated requests using the AWS CLI:
+
+1. Note the _CognitoUserPoolId_ and _CognitoUserPoolClientId_ parameters
+   displayed in the _Outputs_ tab for the stack, and take note of cognito user
+   email and password.
```
[the remark this answers is not in the document]
Comment (matteofigus) [reply docs/USER_GUIDE.md]: Good idea, done

File: docs/USER_GUIDE.md [hunk 7]
```diff
@@ -279,9 +286,35 @@ To add more users to the application:
...
+     --user-pool-id <user-pool-id> \
+     --client-id <user-pool-client-id> \
+     --auth-flow ADMIN_NO_SRP_AUTH \
+     --auth-parameters '{"USERNAME":"<user-email>","PASSWORD":"<your-password"}'
+   ```
+
+3. Take note of the `IdToken` from the previous command. Now you can run an
+   authenticated request to the API. Take note of the API base url from the
+   _ApiUrl_ in the _Outputs_ tab for the CloudFormation stack and use it to
+   build a url. For instance, this should show the Deletion queue:
```
[the remark this answers is not in the document]
Comment (matteofigus) [reply docs/USER_GUIDE.md]: Done

**Comment by HieronymusLex** [comment]
LGTM but I don't think we can merge this till after #200 as it looks like #200 is provisionally included in v0.11?

**Review by ctd — APPROVED** [review]

**Pull request merged**